plugin/wakatime.vim

```diff
@@ -6,7 +6,7 @@
 " Website:     https://wakatime.com/
 " ============================================================================
 
-let s:VERSION = '9.0.1'
+let s:VERSION = '9.1.0'
```

Comment:
Let's not mix version bumps with code changes.

Reply:
I did it to test locally and forgot to rollback :(